code/game/objects/items/devices/traitordevices.dm

```diff
@@ -269,19 +269,93 @@ effective or pretty fucking useless.
 
 /obj/item/storage/toolbox/emergency/turret/PopulateContents()
 	new /obj/item/screwdriver(src)
-	new /obj/item/wrench(src)
+	new /obj/item/wrench/combat(src)
 	new /obj/item/weldingtool(src)
 	new /obj/item/crowbar(src)
 	new /obj/item/analyzer(src)
 	new /obj/item/wirecutters(src)
 
-/obj/item/storage/toolbox/emergency/turret/attackby(obj/item/I, mob/living/user, params)
-	if(I.tool_behaviour == TOOL_WRENCH && user.combat_mode)
-		user.visible_message(span_danger("[user] bashes [src] with [I]!"), \
-			span_danger("You bash [src] with [I]!"), null, COMBAT_MESSAGE_RANGE)
+/obj/item/storage/toolbox/emergency/turret/attackby(obj/item/attacking_item, mob/living/user, params)
+	if(istype(attacking_item, /obj/item/wrench/combat) && attacking_item.toolspeed && user.combat_mode && attacking_item.use_tool(src, user, 2 SECONDS, volume = 50))
```

Comment:
you should be allowed to do this with any wrench but slower if it's not a combat wrench

Reply:
i dont want to let people facetank the bullets to steal the traitor item for themselves

Reply:
my concern is the traitor losing his special wrench and being unable to use his toolbox as a result